src/pallene/typechecker.lua

```diff
@@ -974,7 +974,7 @@ function Checker:check_exp_synthesize(exp)
         -- We check the child expression with verify instead of synthesize because Pallene cases
         -- also act as type annotations for things like empty array literals: ({} as {value}).
         -- However, this means that the call to verify almost always inserts a redundant cast node.
-        -- To keep the --dump-checker output clean, we get rid of it.  By the way, the Pallene to
+        -- To keep the --dump-typechecker output clean, we get rid of it.  By the way, the Pallene to
```

Comment:
I didn't see if/where this flag was implemented. There was another reference to a “--dump compiler option” in a comment in driver.lua (line 51). But I don't see any --dump flags implemented in pallenec.lua. Were they replaced by `--emit-c` and `--print-ir`?

Reply:
Nice catch. The dump flag was removed in #536. I'll fix that comment later.